src/ui/index.js

```diff
@@ -83,23 +87,29 @@ export default async (kbnServer, server, config) => {
     };
   }
 
-  function viewAppWithPayload(app, payload) {
+  async function viewAppWithPayload(app, payload, acceptLanguages) {
+    const languageTags = getLanguageTags(acceptLanguages);
+    const requestedTranslations = await server.plugins.i18n.getTranslations.apply(this, languageTags);
```

Comment:
That was my bad. I had an issue on the API when i defined the rest operator. Fixed now.